libcob/common.c

```diff
@@ -531,6 +531,7 @@ static struct config_enum beepopts[] = {{"FLASH", "1"}, {"SPEAKER", "2"}, {"FALS
 static struct config_enum timeopts[] = {{"0", "1000"}, {"1", "100"}, {"2", "10"}, {"3", "1"}, {NULL, NULL}};
 static struct config_enum syncopts[] = {{"P", "1"}, {NULL, NULL}};
 static struct config_enum varseqopts[] = {{"0", "0"}, {"1", "1"}, {"2", "2"}, {"3", "3"}, {NULL, NULL}};
+static struct config_enum sighdlrregopts[] = {{"0", "0"}, {"1", "1"}, {"2", "2"}, {NULL, NULL}};
```

Comment:
If you can com up with a nice label for those values (maybe "default", "available", "ignore"?) then lpase use them in this tuple.

Reply:
I have added the labels as you suggested, but wouldn't it create confusion because we use `getenv` and directly check if `COB_SIGNAL_REGIME` is either 0,1 or 2? What I mean is for example `COB_SIGNAL_REGIME=default` wouldn't work as expected, which is the intended use case if I am understanding the configuration setup correctly.